bciers/apps/reporting/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Install dependencies only when needed
+FROM docker.io/node:lts-alpine as deps
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /usr/src/app
+COPY ./package*.json ./
+RUN yarn install --frozen-lockfile --production==false
+
+# Production image, copy all the files and run next
+FROM docker.io/node:lts-alpine as runner
```

Comment:
Same here

Reply:
See above in eacb7a1c1cd01b35930228a2b93d9528218a7f17.